test/test_image_classification/test_densenet121.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+path_apphub='../apphub/'
```

Comment:
instead of using "../apphub/",  let's automatically infer the path like this 
 https://github.com/fastestimator/fastestimator/blob/r0.2/test/tests.sh

then we don't need to go to the test folder to execute the bash